client/src/App.js

```diff
@@ -59,23 +63,86 @@ class App extends Component {
         ProxyWalletAbi,
         ProxyWalletAddress,
       );
-
+     
       const FutureTokenAddress = map.dev.FutureToken.toString();
       const FutureTokenAbi = FutureToken.abi;
       const FutureTokenInstance = new web3.eth.Contract(
         FutureTokenAbi,
         FutureTokenAddress,
       );
 
+      // cUSDC address and ABI
+      // I've moved this up so that I can call the cUsdcAddress lower
+      const cUsdcAddress = config.cUsdcAddress;
+      const cUsdcAbi = config.cUsdcAbi;
+      const cUsdcContract = new web3.eth.Contract(
+        cUsdcAbi, 
+        cUsdcAddress,
+      );
+
```

Comment:
no prob - later we will move all of this contract interaction and initialization stuff out to a separate file